recipes/quantlib-python/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "QuantLib-python" %}
+{% set version = "1.26" %}
+{% set sha256 = "84838a1d0ad3994a195646c04e9c85bfc9314b8b3987c26853ea5d645fb62a8e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/lballabio/QuantLib-SWIG/releases/download/QuantLib-SWIG-v{{ version }}/QuantLib-SWIG-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - autoconf  # [unix]
+    - automake  # [unix]
+    - make  # [unix]
+    - swig
+    - boost-cpp
+    - quantlib
+  host:
+    - python
+    - pip
+    - quantlib  # for quantlib-config
+
+  run:
+    - python
+    - quantlib
+    - boost-cpp
```

Comment:
```suggestion
```
Boost does not belong in this run requirements section because your SWIG wrapper doesn't link directly to any boost shared library (checked with ldd). Boost is a dependency of quantlib not quantlib-python.

```
(base) bash-4.4$ ldd _QuantLib.cpython-39-x86_64-linux-gnu.so 
./_QuantLib.cpython-39-x86_64-linux-gnu.so: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.26' not found (required by ./_QuantLib.cpython-39-x86_64-linux-gnu.so)
	linux-vdso.so.1 (0x00007ffe855e3000)
	libQuantLib.so.0 => not found
	libstdc++.so.6 => /lib64/libstdc++.so.6 (0x00007fef08af0000)
	libm.so.6 => /lib64/libm.so.6 (0x00007fef0876e000)
	libgcc_s.so.1 => /lib64/libgcc_s.so.1 (0x00007fef08556000)
	libc.so.6 => /lib64/libc.so.6 (0x00007fef08191000)
	/lib64/ld-linux-x86-64.so.2 (0x00007fef0a51a000)
```

Please also check that boost-cpp is even required to build the SWIG python wrapper of quantlib. If boost isn't mentioned in the quantlib headers then it shouldn't be needed to compile the SWIG python wrapper.